bondible-app/src/pages/MainPage.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles'
+import homePage from '../images/homePage.png'
+
+const useStyles = makeStyles({
+    imgWrapper: {
+      position: 'relative',
+      display: 'flex',
+      justifyContent: 'center',
+      width: 'auto'
+    },
+    img: {
+      position: 'relative',
+      top: '200px',
+      maxWidth: '100%',
+      height: 'auto'
+    }
+  });
+
+  const MainPage: React.FC = (): any => {
```

Comment:
Is there a specific reason to use React.FC and declare the return type here?

Reply:
Removed type any but reckon declaring as Functional Component is worth its time?

Reply:
I'm reading online that React.FC isn't necessary, but it's also not deprecated. We're using it in some places but not in others 🤣 so we should decide whether we use it or not for the sake of consistency.

Reply:
Lets sack it off then, we know its an FC. Was only declaring it because TYPESCRIPT